shared-version.properties

```diff
@@ -1,3 +1,3 @@
 // ALSO CHANGE integration-common version in src/main/resources/create-gradle-airgap-script.ft
```

Comment:
Do we need to update the integration-common version in the airgap script as the comment suggests? 

1. Based on my understanding of the library release order [here](https://sig-confluence.internal.synopsys.com/pages/viewpage.action?spaceKey=integration&title=Library+Release+Order), it is not required? If so, we should remove this comment? 
2. Maybe I am missing something and bumping blackduck-common version should always mean bumping up the integration-common version as well -- in which case we should update the script in the PR

Reply:
I think it's mostly a mention of potential necessary overrides for the airgap build script? I'm not fully sure, I know we have followed that comment before and upgraded the version of integration-common. Given that the airgap file already references the latest version of integration-common and that the change here only impacts blackduck-common I'd favor making no further changes at this point.